package.json

```diff
@@ -14,9 +14,13 @@
     "dev": "npm run dev-serve | npm run dev-build",
     "dev-serve": "NODE_ENV=development node server/index.js",
     "dev-build": "webpack-dev-server --progress --history-api-fallback --hot --content-base build --port 8090 --config webpack.dev.config",
+    "prod": "npm run prod-serve | npm run prod-build",
```

Comment:
i'm not seeing any difference functionally, but running prod-build first ends up hiding a bunch of output from the build...